libs/core/src/lib/micro-process-flow/components/micro-process-flow-item/micro-process-flow-item.component.ts

```diff
@@ -0,0 +1,41 @@
+import { ChangeDetectionStrategy, Component, ViewEncapsulation, Input, ElementRef, ChangeDetectorRef } from '@angular/core';
+import { MicroProcessFlowItemType } from '../../types';
+
+@Component({
+  selector: 'fd-micro-process-flow-item',
+  templateUrl: './micro-process-flow-item.component.html',
+  encapsulation: ViewEncapsulation.None,
+  changeDetection: ChangeDetectionStrategy.OnPush,
```

Comment:
We don't need this rule `encapsulation: ViewEncapsulation.None,`  here because we don't import any global styles. 
And probably  no need this rule too, because we already have it on parent component.
` changeDetection: ChangeDetectionStrategy.OnPush,`